src/component/FolderInfo.js

```diff
@@ -0,0 +1,49 @@
+﻿import React, { useState, useEffect } from "react";
+import "../style/header.css";
+
+function FolderInfo() {
+  const [folderInfo, setFolderInfo] = useState({
+    ownerName: "",
+    folderName: "",
+    profileImage: "",
+  });
+
+  useEffect(() => {
+    const getFolderInfo = async () => {
+      try {
+        const response = await fetch(
+          "https://bootcamp-api.codeit.kr/api/sample/folder"
+        );
+        if (!response.ok) {
+          throw new Error("error");
+        }
+        const answer = await response.json();
+        setFolderInfo({
+          ownerName: answer.folder.owner.name,
+          folderName: answer.folder.name,
+          profileImage: answer.folder.owner.profileImageSource,
+        });
+      } catch (err) {
+        console.error(err);
+      }
+    };
+    getFolderInfo();
+  }, []);
+
+  return (
+    <div className="folderInfo">
+      <div className="ownerInfo">
+        <img
+          className="folderImage"
+          src={folderInfo.profileImage}
+          alt={folderInfo.ownerName}
+        />
+        <p className="ownerName">@{folderInfo.ownerName}</p>
+      </div>
+
+      <p className="folderName">{folderInfo.folderName}</p>
+    </div>
+  );
```

Comment:
```suggestion
  if (!folderInfo.profileImage || !folderInfo.ownerName || 
!folderInfo.folderName) {
    return <></>;
  }

  return (
    <div className="folderInfo">
      <div className="ownerInfo">
        <img
          className="folderImage"
          src={folderInfo.profileImage}
          alt={folderInfo.ownerName}
        />
        <p className="ownerName">@{folderInfo.ownerName}</p>
      </div>

      <p className="folderName">{folderInfo.folderName}</p>
    </div>
  );
```

최초 렌더링 시점(마운트)에는 folderInfo 값이 모두 빈스트링으로 초기화되어 있기 때문에 값이 비어있는 케이스를 대응해주면 에러도 예방할 수 있고 로딩 흐름도 제어할 수 있기 때문에 더 좋습니다.